packages/playground/tests/frontend_selenium/tests/Farms/test_node.py

```diff
@@ -0,0 +1,285 @@
+from utils.utils import get_email, generate_inavalid_gateway, generate_inavalid_ip, generate_ip, generate_leters, generate_string, get_node_seed, randomize_public_ipv4, valid_amount
+from pages.node import NodePage
+from utils.grid_proxy import GridProxy
+from pages.dashboard import DashboardPage
+from datetime import datetime
+import random
+import math
+import time
+
+#  Time required for the run (12 cases) is approximately 3 minutes.
+
+def before_test_setup(browser):
+    node_page = NodePage(browser)
+    grid_proxy = GridProxy(browser)
+    dashboard_page = DashboardPage(browser)
+    password = generate_string()
+    dashboard_page.open_and_load()
+    dashboard_page.import_account(get_node_seed())
+    dashboard_page.click_button(dashboard_page.connect_your_wallet(get_email(), password))
+    node_page.navigate()
+    return node_page, grid_proxy
+
+
+def test_node_page(browser):
+    """
+      Test Case: TC1216 - Navigate to farm node
+      Steps:
+          - Navigate to the dashboard.
+          - Login into an account (with node).
+          - Click on Farm from side menu.
+      Result: User should be navigated to Farm Nodes page and see all his node.
+    """
+    node_page, grid_proxy = before_test_setup(browser)
+    nodes = grid_proxy.get_twin_node(str(node_page.twin_id))
+    for node in nodes:
+        assert str(node['nodeId']) in browser.page_source
+
+
+# def test_search_node(browser):
+#     """
+#       Test Case: TC1217 - Search node
+#       Steps:
```

Comment:
Do we comment this for a reason?

Reply:
Node search was removed from new UI, I comment the test until it come back.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/ed29b295-fd31-4e0e-93c9-d4f93d7bcc8f)